qui/tray/domains.py

```diff
@@ -253,7 +253,14 @@ def update_domain_item(self, _, vm_path):
             'notify-send',
             "Domain %s is %s" % (vm['name'], str(vm['state']).lower())
         ])
-        self.tray_menu.add(domain_item)
+
+        position = 0
+        for i in self.tray_menu:
+            if i.vm['name'] > vm['name']:
+                break
+            position += 1
+            
```

Comment:
Code style: please remove trailing spaces

Reply:
done
I forgot to post the additional space in my IDE